service_container/3.3-di-changes.rst

```diff
@@ -373,7 +373,8 @@ The third big change is that, in a new Symfony 3.3 project, your controllers are
         // app/config/services.php
 
         // ...
-
+        
+        $definition->addTag('controller.service_arguments');
         $this->registerClasses($definition, 'AppBundle\\Controller\\', '../../src/AppBundle/Controller/*');
```

Comment:
It does not matter, but would it look better if we add the tag after the classes registration? i.e the same natual order that `yaml` and `xml` formats.

Reply:
But then the tag wouldn't be picked up inside `registerClasses()` where the past `Definition` instance is used as a prototype.